bin/get-teamcity-artifact-url

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env node
+
+// This script will return a url of an artifact generated in the latest project build for the specified branch.
+// eg: node bin/get-teamcity-artifact-url --artifact="translate/calypso-strings.pot"
+
+const parsedArgs = process.argv
+	.slice( 2 )
+	.filter( ( arg ) => arg.startsWith( '--' ) )
+	.reduce( ( accumulator, arg ) => {
+		const [ key, value ] = arg.slice( 2 ).split( '=' );
+		accumulator[ key ] = value;
+
+		return accumulator;
+	}, {} );
+
+const args = {
+	branch: 'trunk',
+	buildType: 'calypso_Translate',
+	artifact: '',
+	...parsedArgs,
+};
+
+( async function () {
+	if ( args.artifact ) {
```

Comment:
Definitely nitpicky, but I would rephrase this as:

```js
if ( ! args.artifact ) {
  process.exit( 1 )
}

const url = ...
```

Reply:
With the switch to `yargs` and the use of `.required()`, it's no longer needed to do the check.